pages/docs/concepts/consumer.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Consumer
+weight: 4
+---
+
+## What is a Consumer?
+In an Event Driven Architecture (EDA), a consumer is a component of an application or messaging system that listens for a particular event from a broker and takes the necessary action in response.
+
+## Why do we need Consumers?
```

Comment:
```suggestion
## What is the purpose of Consumers?
```

This is the outline all **Concept** Docs will have: `What is the purpose of _____?`

Reply:
Okay, I will correct that asap!

Reply:
don't hate but i changed my mind :D haha!

Let's stick with your original one, I went ahead and fixed it :D

Reply:
Alrighty!